mautic/mautic
# Fix csv header with 'file' column issue while contact import

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅ 
| Related user documentation PR URL      | ❌ 
| Related developer documentation PR URL | ❌ 
| Issue(s) addressed                     | ❌ 



#### Description:
Having "file" as the name of a column in a CSV breaks the contact import. This PR address the issue by prefixing the CSV headers to avoid form rendering errors in contact import.
Error message:

> Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 1 passed to Symfony\Bundle\FrameworkBundle\Templating\Helper\FormHelper::widget() must be an instance of Symfony\Component\Form\FormView, null given, called in themes/system/MauticLeadBundle/Import/upload_form.html.php on line 44" at vendor/symfony/framework-bundle/Templating/Helper/FormHelper.php line 136

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create CSV with 'file' header and some contacts
3. Upload the CSV to import contacts
4. Mapping screen should be visible and contact import should be successful

Creating this PR in favour of https://github.com/mautic/mautic/pull/10489 to avoid resolving conflicts from earlier base.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10823"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
